snuba/consumers/consumer_builder.py

```diff
@@ -54,6 +57,50 @@ class ProcessingParameters:
     output_block_size: Optional[int]
 
 
+def verify_single_topic(storages: Sequence[WritableTableStorage]) -> str:
```

Comment:
Not sure I understand why this function returns a string. It verifies 3 topics - the input one, the commit log one, and the replacement one. Why does it need to return one of them?

This is similar to the code that already exists in the multistorage implementation. I would suggest you avoid making a second copy of that code by either:
1) calling this function from that implementation as well, or:
2) calling that existing implementation from here then later moving it over

I think 2 could be a good idea, so you can make smaller incremental changes.

Reply:
The reason why it's returning the string (which is the raw logical topic) is because I thought it'd be easy/cleaner to just return that from this function, which already computes `default_topics`. Otherwise I would just have to compute `default_topics` again and get the topic again outside of this `verify_single_topic` function. I can do that too, this was just for cleaniness/not repeating computations.